main.js

```diff
@@ -1,53 +1,84 @@
-let books = [];
-let counter = 0;
-const bookList = document.getElementById('bookList');
-const savedBooks = localStorage.getItem('savedBooks');
-
-function display({ title, author, id }) {
-  const li = document.createElement('li');
-  const titleP = document.createElement('p');
-  const authorP = document.createElement('p');
-  const button = document.createElement('button');
-
-  titleP.innerHTML = title;
-  authorP.innerHTML = author;
-  button.innerHTML = 'remove';
-  button.className = 'remove';
-  button.addEventListener('click', () => {
-    books = books.filter((obj) => obj.id !== id);
-    bookList.innerHTML = '';
-    books.forEach((book) => {
+// eslint-disable-next-line max-classes-per-file
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class UI {
+  constructor() {
+    this.savedBook = [];
+    this.bookList = document.getElementById('bookList');
+    this.parseBook();
+  }
+
```

Comment:
- [ ] The functionality to add books is broken, it throws this error in the console: Uncaught TypeError: Cannot read property 'forEach' of null at UI.displayBooks. Kindly check and fix it.